### ISSUE TITLE
Unable to register package source

### ISSUE THREAD

**Issue by ethanbergstrom** [ethanbergstrom/Chocolatier#11]
Hello @ethanbergstrom, 

I saw that there was a new version 1.2.0 of Chocolatier. I don't know if anyone has tested adding their own repository via "register-packagesource" commands, but I get an error in the 'name' parameter (see below) 

PS C:\Windows\system32> Register-PackageSource -name TEST -location https://myrepo/nuget -provider Chocolatier -Trusted 

WARNING: MSG:UnableToDownload «https://go.microsoft.com/fwlink/?LinkID=627338&clcid=0x409» «»
WARNING: Unable to download the list of available providers. Check your internet connection.
Register-PackageSource : Cannot bind argument to parameter 'name' because it is an empty string.
At line:1 char:1
+ Register-PackageSource -name TEST -location https://myrepo ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (Microsoft.Power...erPackageSource:RegisterPackageSource) [Register-PackageSource], Exception
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,New-PackageSource,Microsoft.PowerShell.PackageManagement.Cmdlets.RegisterPackageSour 
   ce

Can you confirm. Do you have that the same problem? 
Is this a mistake on the side of the new provider Chocolatier 1.2.0 ?

This problem does not occur in a e.g. 1.0.4 Chocolatier version.

_Originally posted by @Gregoorio in https://github.com/jianyunt/ChocolateyGet/pull/15#issuecomment-650182485_

**Comment by ethanbergstrom** [comment]
Hey @Gregoorio,

I'm not able to reproduce the issue you're seeing. What version of PowerShell are you using?

**Comment by Gregoorio** [comment]
@ethanbergstrom,

I'm using the 5.1 powershell version. In version 1.0.4 Chocolatier I used the same version and registration of the repository via (register-packagesource) command it is works.

**Comment by ethanbergstrom** [comment]
Could you paste the entire terminal output, including the registration command, in this thread?

**Comment by Gregoorio** [comment]
Hi  @ethanbergstrom,

I've used command: 
Register-PackageSource -name TEST -location https://myrepo.hs/nuget -provider Chocolatier -Trusted 

And this is error: 

WARNING: MSG:UnableToDownload «https://go.microsoft.com/fwlink/?LinkID=627338&clcid=0x409» «»
WARNING: Unable to download the list of available providers. Check your internet connection.
Register-PackageSource : Cannot bind argument to parameter 'name' because it is an empty string.
At line:1 char:1
+ Register-PackageSource -name TEST -location https://myrepo.hs. ...+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~+ CategoryInfo          : InvalidData: (Microsoft.Power...erPackageSource:RegisterPackageSource) [Register-PackageSource], Exception + FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,New-PackageSource,Microsoft.PowerShell.PackageManagement.Cmdlets.RegisterPackageSource

I saw problem in the parameter 'name' but I don't know what is the matter

"Cannot bind argument to parameter 'name' because it is an empty string."

**Comment by ethanbergstrom** [comment]
I just tried it in a clean Windows Sandbox environment, and I'm not seeing that issue:
![image](https://user-images.githubusercontent.com/13603162/85924498-5f13fe00-b858-11ea-8390-05e613c38b8a.png)

**Comment by ethanbergstrom** [comment]
Could you confirm how you're installing the package provider? From the output, it looks like your machine isn't connected to the internet. Are you mirroring the package provider module to an internal NuGet repository?

**Comment by Gregoorio** [comment]
Yes, you are right. My machine has no internet. 
I have download a Chocolatier package from another machine that has the Internet and copy folders to a machine without the Internet. 
I have an external nugget repository by (nuget server). 
From a machine that doesn't have internet I register my nuget repository.  In version 1.0.4 of Provider Chocolatier it works. However in version 1.2.0 I have a problem with the 'name' parameter.

**Comment by ethanbergstrom** [comment]
I'm not sure what your issue may be then, other than some sort of issue in your copy process to the disconnected machine.

I just tried copying the module into a Windows Sandbox instance without any internet connection, and it worked after I set the execution policy to RemoteSigned. 

![image](https://user-images.githubusercontent.com/13603162/85933411-4c231d00-b89c-11ea-80a6-2c3e17831e4a.png)

**Comment by Gregoorio** [comment]
Hi ethanbergstrom,

I copied the files from chocolatier provider 1.0.4 into the machine without the internet connection. There's one thing that has been bothering me,  so I have deregistered my repository and copied the provider 1.2.0 files. Once again I have tried to register and the registered correctly. 
I don't know what was the reason for the problem before. Currently the registered repository works in provider version 1.2.0.
Sorry for the confusion and thank you for your analysis.
